### ISSUE TITLE
zigbee2mqtt script does not setup usb passthru

### ISSUE THREAD

**Issue by dreed47** [tteck/Proxmox#2360]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

After running the script to install the USB coordinator device that im using does not show in the newly created CT when I use the ls -l /dev/serial/by-id command

### Which Linux distribution are you employing?

Debian 11

### Please provide detailed steps to reproduce the issue.

Just run the script while the desired USB coordinator is plugged into the proxmox machine.

**Comment by tteck** [comment]
Was the LXC created Privileged?

**Comment by dreed47** [comment]
yes

**Comment by dreed47** [comment]
<img width="440" alt="ddd" src="https://github.com/tteck/Proxmox/assets/212698/bfa2ddf7-4513-439d-afa5-fe2a4ea8ac25">

**Comment by tteck** [comment]
Run this command within the Proxmox shell (change <CTID> to your ID) `cat /etc/pve/lxc/<CTID>.conf` and share the results.

**Comment by dreed47** [comment]
```#<div align='center'><a href='https%3A//Helper-Scripts.com' target='_blank' rel='noopener noreferrer'><img src='https%3A//raw.githubusercontent.com/tteck/Proxmox/main/misc/images/logo-81x112.png'/></a>
#
#  # Zigbee2MQTT LXC
#
#  <a href='https%3A//ko-fi.com/D1D7EP4GF'><img src='https%3A//img.shields.io/badge/&#x2615;-Buy me a coffee-blue' /></a>
#  </div>
arch: amd64
cores: 2
features: nesting=1
hostname: zigbee2mqtt
memory: 1024
net0: name=eth0,bridge=vmbr0,hwaddr=FE:9B:ED:A9:24:F1,ip=dhcp,type=veth
onboot: 1
ostype: debian
rootfs: local-lvm:vm-101-disk-0,size=4G
swap: 512
tags: proxmox-helper-scripts
lxc.cgroup2.devices.allow: a
lxc.cap.drop: 
lxc.cgroup2.devices.allow: c 188:* rwm
lxc.cgroup2.devices.allow: c 189:* rwm
lxc.mount.entry: /dev/serial/by-id  dev/serial/by-id  none bind,optional,create=dir
lxc.mount.entry: /dev/ttyUSB0       dev/ttyUSB0       none bind,optional,create=file
lxc.mount.entry: /dev/ttyUSB1       dev/ttyUSB1       none bind,optional,create=file
lxc.mount.entry: /dev/ttyACM0       dev/ttyACM0       none bind,optional,create=file
lxc.mount.entry: /dev/ttyACM1       dev/ttyACM1       none bind,optional,create=file```

**Comment by tteck** [comment]
USB passthrough is enabled

**Comment by dreed47** [comment]
```root@proxmox:~# lsusb
Bus 004 Device 002: ID 059f:10bc LaCie, Ltd Mobile Drive
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 002: ID 0781:5581 SanDisk Corp. Ultra
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 011: ID 10c4:ea60 Silicon Labs CP210x UART Bridge
Bus 001 Device 003: ID 8087:0aaa Intel Corp. Bluetooth 9460/9560 Jefferson Peak (JfP)
Bus 001 Device 010: ID 1a86:7523 QinHeng Electronics CH340 serial converter
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
root@proxmox:~# ```

**Comment by dreed47** [comment]
im trying to expose either 1a86:7523 or 10c4:ea60 but neither show in the CT

**Comment by tteck** [comment]
Have you tried rebooting the LXC?

**Comment by dreed47** [comment]
yeah...let me try again....

same result

```
root@zigbee2mqtt:~# ls -l /dev/serial/by-id
total 0
root@zigbee2mqtt:~# 
```

**Comment by tteck** [comment]
I'm creating a new Zigbee2MQTT LXC to test...

**Comment by tteck** [comment]
```
 _____   _       __             ___   __  _______  ____________
/__  /  (_)___ _/ /_  ___  ___ |__ \ /  |/  / __ \/_  __/_  __/
  / /  / / __  / __ \/ _ \/ _ \__/ // /|_/ / / / / / /   / /   
 / /__/ / /_/ / /_/ /  __/  __/ __// /  / / /_/ / / /   / /    
/____/_/\__, /_.___/\___/\___/____/_/  /_/\___\_\/_/   /_/     
       /____/ 🐝
 
Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 0
Using Root Password: Automatic Login
Container ID: 111
Using Hostname: zigbee2mqtt
Using Disk Size: 4
Allocated Cores: 2
Allocated RAM: 1024
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: 192.168.86.110
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Zigbee2MQTT LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 111 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.253 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.113.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up Node.js Repository
 ✓ Installed Node.js
 ✓ Set up Zigbee2MQTT Repository
Switch to Edge/dev branch? (y/N) 
 ✓ Installed Zigbee2MQTT
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!
```
```
root@zigbee2mqtt:~# ls -l /dev/serial/by-id
total 0
lrwxrwxrwx 1 root root 13 Jan  4 15:33 usb-Silicon_Labs_CP2102N_USB_to_UART_Bridge_Controller_2e893de28e93eb11842d214f3d943674-if00-port0 -> ../../ttyUSB0
root@zigbee2mqtt:~# 
```

**Comment by tteck** [comment]
Why are you creating it with Debian 11? 
Still using PVE 7?
what is showing in the `/dev` directory?

**Comment by dreed47** [comment]
```
You are currently using Proxmox VE 7 (EOL 2024-07),      │ 
            │ refrain from creating Debian 12 LXCs.                    │ 
             │ Default distribution for Zigbee2MQTT LXC is debian 12 
```

**Comment by dreed47** [comment]
```
Creating a Zigbee2MQTT LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 - Creating LXC Container  -unable to create CT 101 - unsupported debian version '12.2'
\bash: line 182: exit: A problem occured while trying to create container.: numeric argument required
root@proxmox:~#-

```

**Comment by tteck** [comment]
Time for PVE 8

I'm creating a new Zigbee2MQTT LXC Debian 11 to test...

**Comment by tteck** [comment]
```
 _____   _       __             ___   __  _______  ____________
/__  /  (_)___ _/ /_  ___  ___ |__ \ /  |/  / __ \/_  __/_  __/
  / /  / / __  / __ \/ _ \/ _ \__/ // /|_/ / / / / / /   / /   
 / /__/ / /_/ / /_/ /  __/  __/ __// /  / / /_/ / / /   / /    
/____/_/\__, /_.___/\___/\___/____/_/  /_/\___\_\/_/   /_/     
       /____/ 🐝
 
Using Advanced Settings
Using Distribution: debian
Using debian Version: 11
Using Container Type: 0
Using Root Password: Automatic Login
Container ID: 111
Using Hostname: zigbee2mqtt
Using Disk Size: 4
Allocated Cores: 2
Allocated RAM: 1024
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a Zigbee2MQTT LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 111 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.253 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.113.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up Node.js Repository
 ✓ Installed Node.js
 ✓ Set up Zigbee2MQTT Repository
Switch to Edge/dev branch? (y/N) 
 ✓ Installed Zigbee2MQTT
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!
```
Same result

```
root@zigbee2mqtt:~# ls -l /dev/serial/by-id
total 0
lrwxrwxrwx 1 root root 13 Jan  4 15:33 usb-Silicon_Labs_CP2102N_USB_to_UART_Bridge_Controller_2e893de28e93eb11842d214f3d943674-if00-port0 -> ../../ttyUSB0
root@zigbee2mqtt:~# 
```

**Comment by dreed47** [comment]
I just wiped mine out and started over with debian 11 but I get the same results .. no usb device in the container :(  Am I going to need to upgrade proxmox to be able to use this script?

**Comment by dreed47** [comment]
I do get an error that says it cant find the internet when it installs

**Comment by tteck** [comment]
What's the output of `ls /dev` in the z2m container?

**Comment by dreed47** [comment]
let me check....

this is the internet error I get 

```
Creating a Zigbee2MQTT LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 103 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 2603:6011:2806:fbb5:9426:68ff:fec5:1815 
 ✗ Internet NOT Connected
Would you like to continue anyway? <y/N> 
```

**Comment by tteck** [comment]
No IPv4 address showing, only IPv6

**Comment by dreed47** [comment]
```
root@zigbee2mqtt:~# ls -l /dev             
total 0
crw--w---- 1 root   tty    136, 0 Jan 18 13:16 console
lrwxrwxrwx 1 root   root       11 Jan 18 13:15 core -> /proc/kcore
lrwxrwxrwx 1 root   root       13 Jan 18 13:15 fd -> /proc/self/fd
crw-rw-rw- 1 root   root     1, 7 Jan 18 13:15 full
drwxr-xr-x 2 root   root        0 Jan 18 13:15 hugepages
lrwxrwxrwx 1 root   root       12 Jan 18 13:15 initctl -> /run/initctl
lrwxrwxrwx 1 root   root       28 Jan 18 13:15 log -> /run/systemd/journal/dev-log
drwxrwxrwt 2 root   root       40 Jan 18 13:15 mqueue
crw-rw-rw- 1 root   root     1, 3 Jan 18 13:15 null
crw-rw-rw- 1 root   root     5, 2 Jan 18 13:18 ptmx
drwxr-xr-x 2 root   root        0 Jan 18 13:15 pts
crw-rw-rw- 1 root   root     1, 8 Jan 18 13:15 random
drwxr-xr-x 3 root   root       60 Jan 18 13:15 serial
drwxrwxrwt 2 root   root       40 Jan 18 13:15 shm
lrwxrwxrwx 1 root   root       15 Jan 18 13:15 stderr -> /proc/self/fd/2
lrwxrwxrwx 1 root   root       15 Jan 18 13:15 stdin -> /proc/self/fd/0
lrwxrwxrwx 1 root   root       15 Jan 18 13:15 stdout -> /proc/self/fd/1
crw-rw-rw- 1 root   root     5, 0 Jan 18 13:15 tty
crw------- 1 root   tty    136, 1 Jan 18 13:18 tty1
crw--w---- 1 root   tty    136, 2 Jan 18 13:15 tty2
---------- 1 root   root        0 Jan 18 13:15 ttyACM0
---------- 1 root   root        0 Jan 18 13:15 ttyACM1
crw-rw---- 1 100000 100020 188, 0 Jan 18 12:27 ttyUSB0
crw-rw---- 1 100000 100020 188, 1 Jan 18 12:29 ttyUSB1
crw-rw-rw- 1 root   root     1, 9 Jan 18 13:15 urandom
crw-rw-rw- 1 root   root     1, 5 Jan 18 13:15 zero
```

**Comment by tteck** [comment]
that looks correct, still nothing in `ls -l /dev/serial/by-id`?

**Comment by dreed47** [comment]
no

**Comment by tteck** [comment]
Reboot Proxmox

**Comment by dreed47** [comment]
ok

**Comment by dreed47** [comment]
done but still nothing in /dev/serial/by-id

**Comment by dreed47** [comment]
im rebooting the machine now

**Comment by tteck** [comment]
When you reboot Proxmox, that is the machine.

**Comment by dreed47** [comment]
no I know but proxmox is install on my intel NUC computer...so I first rebooted proxmox in their UI but then rebooted the NUC ...but...end result is the same....no USB devices

**Comment by tteck** [comment]
I have no idea, sorry.

**Comment by dreed47** [comment]
this is what /dev looks like in the container
```
root@zigbee2mqtt:~# ls -l /dev
total 0
crw--w---- 1 root tty     136, 0 Jan 18 13:26 console
lrwxrwxrwx 1 root root        11 Jan 18 13:26 core -> /proc/kcore
lrwxrwxrwx 1 root root        13 Jan 18 13:26 fd -> /proc/self/fd
crw-rw-rw- 1 root root      1, 7 Jan 18 13:26 full
drwxr-xr-x 2 root root         0 Jan 18 13:26 hugepages
lrwxrwxrwx 1 root root        12 Jan 18 13:26 initctl -> /run/initctl
lrwxrwxrwx 1 root root        28 Jan 18 13:26 log -> /run/systemd/journal/dev-log
drwxrwxrwt 2 root root        40 Jan 18 13:26 mqueue
crw-rw-rw- 1 root root      1, 3 Jan 18 13:26 null
crw-rw-rw- 1 root root      5, 2 Jan 18 13:31 ptmx
drwxr-xr-x 2 root root         0 Jan 18 13:26 pts
crw-rw-rw- 1 root root      1, 8 Jan 18 13:26 random
drwxr-xr-x 3 root root        60 Jan 18 13:26 serial
drwxrwxrwt 2 root root        40 Jan 18 13:26 shm
lrwxrwxrwx 1 root root        15 Jan 18 13:26 stderr -> /proc/self/fd/2
lrwxrwxrwx 1 root root        15 Jan 18 13:26 stdin -> /proc/self/fd/0
lrwxrwxrwx 1 root root        15 Jan 18 13:26 stdout -> /proc/self/fd/1
crw-rw-rw- 1 root root      5, 0 Jan 18 13:26 tty
crw------- 1 root tty     136, 1 Jan 18 13:31 tty1
crw--w---- 1 root tty     136, 2 Jan 18 13:26 tty2
---------- 1 root root         0 Jan 18 13:26 ttyACM0
---------- 1 root root         0 Jan 18 13:26 ttyACM1
crw-rw---- 1 root dialout 188, 0 Jan 18 13:25 ttyUSB0
crw-rw---- 1 root dialout 188, 1 Jan 18 13:25 ttyUSB1
crw-rw-rw- 1 root root      1, 9 Jan 18 13:26 urandom
crw-rw-rw- 1 root root      1, 5 Jan 18 13:26 zero
root@zigbee2mqtt:~# 
```

**Comment by tteck** [comment]
What does `ls -l /dev/serial/by-id` show in the Proxmox shell?

**Comment by dreed47** [comment]
in host 
```
root@proxmox:~# ls -l /dev/serial/by-path/
total 0
lrwxrwxrwx 1 root root 13 Jan 18 13:25 pci-0000:00:14.0-usb-0:1:1.0-port0 -> ../../ttyUSB0
lrwxrwxrwx 1 root root 13 Jan 18 13:25 pci-0000:00:14.0-usb-0:4:1.0-port0 -> ../../ttyUSB1
root@proxmox:~# 
```

**Comment by dreed47** [comment]
I dont see a by-id only by-path

**Comment by tteck** [comment]
The issue is not with the container then.

**Comment by dreed47** [comment]
also on host
```
root@proxmox:~# lsusb
Bus 004 Device 002: ID 059f:10bc LaCie, Ltd Mobile Drive
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 002: ID 0781:5581 SanDisk Corp. Ultra
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 003: ID 10c4:ea60 Silicon Labs CP210x UART Bridge
Bus 001 Device 004: ID 8087:0aaa Intel Corp. Bluetooth 9460/9560 Jefferson Peak (JfP)
Bus 001 Device 002: ID 1a86:7523 QinHeng Electronics CH340 serial converter
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
root@proxmox:~# 
```

**Comment by tteck** [comment]
Before I went any further, and seeing that the CTID is 101, I would do a fresh install of Proxmox 8 and start again.

**Comment by dreed47** [comment]
ok, thanks for your help.  I don't have an easy sandbox to do a fresh install of 8, is it feasible to upgrade 7 without a lot of headaches?

**Comment by tteck** [comment]
> I don't have an easy sandbox to do a fresh install of 8

What does this mean?

**Comment by dreed47** [comment]
a place to install it?  I have 7 installed on my NUC computer

**Comment by tteck** [comment]
Overwrite PVE 7 with PVE8
What is the exact version of your PVE 7?

**Comment by dreed47** [comment]
7.4-3

**Comment by tteck** [comment]
Why is it so old? Have you not ever run an update?

**Comment by dreed47** [comment]
:). yeah.. I installed it and have plex a a couple other stuff installed from a while ago and never upgraded

**Comment by tteck** [comment]
Execute `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/post-pve-install.sh)"` within the Proxmox shell

**Comment by dreed47** [comment]
I ran it accepting the defaults but im not sure what it did

```
    ____ _    ________   ____             __     ____           __        ____
   / __ \ |  / / ____/  / __ \____  _____/ /_   /  _/___  _____/ /_____ _/ / /
  / /_/ / | / / __/    / /_/ / __ \/ ___/ __/   / // __ \/ ___/ __/ __ `/ / /
 / ____/| |/ / /___   / ____/ /_/ (__  ) /_   _/ // / / (__  ) /_/ /_/ / / /
/_/     |___/_____/  /_/    \____/____/\__/  /___/_/ /_/____/\__/\__,_/_/_/

 ✓ Corrected Proxmox VE Sources
 ✓ Disabled 'pve-enterprise' repository
 ✓ Enabled 'pve-no-subscription' repository
 ✓ Added 'pvetest' repository
 ✓ Disabled high availability
 - Updating Proxmox VE (Patience)...root@proxmox:~# 
```

**Comment by tteck** [comment]
In the shell, `ping google.com`

**Comment by dreed47** [comment]
```
root@proxmox:~# ping google.com
PING google.com (142.250.191.110) 56(84) bytes of data.
64 bytes from ord38s28-in-f14.1e100.net (142.250.191.110): icmp_seq=1 ttl=56 time=22.7 ms
64 bytes from ord38s28-in-f14.1e100.net (142.250.191.110): icmp_seq=2 ttl=56 time=23.1 ms
64 bytes from ord38s28-in-f14.1e100.net (142.250.191.110): icmp_seq=3 ttl=56 time=23.0 ms
64 bytes from ord38s28-in-f14.1e100.net (142.250.191.110): icmp_seq=4 ttl=56 time=22.7 ms
64 bytes from ord38s28-in-f14.1e100.net (142.250.191.110): icmp_seq=5 ttl=56 time=25.3 ms
64 bytes from ord38s28-in-f14.1e100.net (142.250.191.110): icmp_seq=6 ttl=56 time=25.2 ms
^C
--- google.com ping statistics ---
6 packets transmitted, 6 received, 0% packet loss, time 5006ms
rtt min/avg/max/mdev = 22.650/23.664/25.306/1.143 ms
root@proxmox:~# 
```

**Comment by tteck** [comment]
in the shell run
```
apt-get update
apt-get dist-upgrade
```

**Comment by dreed47** [comment]
```
root@proxmox:~# apt-get update
Hit:1 http://ftp.debian.org/debian bullseye InRelease
Hit:2 http://security.debian.org/debian-security bullseye-security InRelease                        
Hit:3 http://ftp.debian.org/debian bullseye-updates InRelease                                       
Hit:4 http://download.proxmox.com/debian/pve bullseye InRelease                                     
Err:5 https://enterprise.proxmox.com/debian/pve bullseye InRelease         
  401  Unauthorized [IP: 144.217.225.162 443]
Get:6 https://apt.iteas.at/iteas buster InRelease [10.7 kB]
Err:6 https://apt.iteas.at/iteas buster InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 23CAE45582EB0928
Reading package lists... Done
E: Failed to fetch https://enterprise.proxmox.com/debian/pve/dists/bullseye/InRelease  401  Unauthorized [IP: 144.217.225.162 443]
E: The repository 'https://enterprise.proxmox.com/debian/pve bullseye InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: GPG error: https://apt.iteas.at/iteas buster InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 23CAE45582EB0928
E: The repository 'https://apt.iteas.at/iteas buster InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
root@proxmox:~# 


```

**Comment by dreed47** [comment]
running apt-get dist-upgrade now

**Comment by dreed47** [comment]
apt-get dist-upgrade  finished

**Comment by tteck** [comment]
Proxmox version now?

**Comment by dreed47** [comment]
7.4-17

**Comment by tteck** [comment]
show this screen from yours

![Screenshot 2024-01-18 2 05 35 PM](https://github.com/tteck/Proxmox/assets/21342670/274dff5a-ff2f-428e-bb1b-8cbe4a7183e0)

**Comment by dreed47** [comment]
<img width="1804" alt="pm1" src="https://github.com/tteck/Proxmox/assets/212698/6fe5c67b-b588-41be-ac07-7b0aa0b9a9ae">

**Comment by tteck** [comment]
run `rm -rf /etc/apt/sources.list.d/iteas.list /etc/apt/sources.list.d/pve-no-subscription.list` in the shell and repost the new screenshot

**Comment by dreed47** [comment]
<img width="1913" alt="pm1" src="https://github.com/tteck/Proxmox/assets/212698/f27b6063-03bc-4079-a140-a1cd92f0e3f2">

**Comment by tteck** [comment]
Much better!

**Comment by dreed47** [comment]
thanks for all your help!!

**Comment by tteck** [comment]
If you want PVE 8, execute
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/pve8-upgrade.sh)"
```
in the shell.

**Comment by dreed47** [comment]
Ill give that a shot now

**Comment by tteck** [comment]
After that completes, rerun
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/post-pve-install.sh)"
```

**Comment by dreed47** [comment]
thx, I ran both scripts

**Comment by tteck** [comment]
Proxmox version now?

**Comment by dreed47** [comment]
8.1.4

**Comment by tteck** [comment]
Now try creating a new Zigbee2MQTT LXC utilizing Debian 12

**Comment by dreed47** [comment]
```
root@zigbee2mqtt:~# ls -l /dev/serial/by-id
total 0
lrwxrwxrwx 1 root root 13 Jan 18 14:39 usb-1a86_USB_Serial-if00-port0 -> ../../ttyUSB0
root@zigbee2mqtt:~# 

```

Looks like that resolved it!!!! Thank you for walking me thru that process!!!!!

**Comment by tteck** [comment]
Glad I could help.
